base/tekton.dev/tasks/cosa-build-baseos.yaml

```diff
@@ -35,7 +35,7 @@ spec:
           echo "Skipping buildfetch from S3 - bucket params not set"
         else
           cosa buildfetch \
-          --url=s3://$(params.bucket-name)/builds \
+          --url=s3://$(params.bucket-name)/$(params.bucket-name)/builds \
```

Comment:
I think we need a "$(params.subfolder)" here, as we're duplicating it as bucket name

Reply:
Yes, ideally we separate the different build streams with subdirs.
However in this case, the bucket-name would have to stay duplicated here, as that's where
`cosa buildupload s3 $(params.bucket-name)/builds` currently unergonomically uploads to (it split's the param and takes the first part as bucket-name).
I'd vote for fixing this later.

Reply:
SGTM